region/brinstar/red/Red Tower.json

```diff
@@ -603,6 +603,28 @@
       ],
       "note": "This is a short climb of only a few tiles."
     },
+    {
+      "link": [2, 1],
+      "name": "Very Deep Stuck X-Ray Climb",
+      "notable": false,
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "canXRayClimb"
+      ],
+      "bypassesDoorShell": true,
+      "note": [
+        "Enter with G-mode direct, back up to between 1 and 6 pixels from the door transition, and activate X-ray to get very deep stuck in the door.",
```

Comment:
A long note for what's likely to be a common strat.

Reply:
Yeah it would be fine to be shortened. Mostly I was thinking we might want to create a tech for it, and the description could be moved there. I wasn't sure because this would be a pretty easy tech for Extreme (aside from the dependence on G-mode), but there is something extra in just knowing that it's possible to do, so having a new tech would be a way to highlight that. On the other hand, G-mode in general involves a lot of knowledge about how it can be used, so that would be an argument for not needing another tech here. In any case, figured a new tech would be a separate issue from this PR

Reply:
Yep, for this PR its fine as is.

GMode very deep stuck is mostly a knowledge check, that should be explained someplace.  And as a knowledge check it could be a tech.  But its really not hard or complex compared to GMode itself so its kinda weird.